### PR TITLE
Add RW1990 support.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,17 +10,20 @@ This library also supports the RW1990 which allows to modify the ID of the iButt
...
-For simplicity, we recommend a 4.7k resistor when no special functionality is needed,
-and a 2.2k resistor otherwise.
+In most cases, the pull-up of the ESP32 is strong enough to power devices that
+are connected to the 1-wire bus. As such, no additional components are needed.
 
 See https://pdfserv.maximintegrated.com/en/an/AN4255.pdf for more information on
 how to power 1-wire devices.
 
+## RW1990 support
+The RW1990 is a device that allows to modify the ID of an iButton. It is not
```
Comment (erikcorry) [inline README.md:20]:
```suggestion
The RW1990 is a device that can modify the ID of an iButton. It is not
```

File: README.md [hunk 2]
```diff
@@ -10,17 +10,20 @@ This library also supports the RW1990 which allows to modify the ID of the iButt
...
+produced by Dallas, but can be found on Chinese websites.
+
+We were not able to reliable change all RW1990 devices we received. Out of
+10 devices, 4 worked without issues, 4 did not allow to change the IDs at all,
```
Comment (erikcorry) [inline README.md:24]:
```suggestion
10 devices, 4 worked without issues, 4 did not allow changing the IDs at all,
```

File: src/ibutton.toit [hunk 3]
```diff
@@ -59,13 +62,56 @@ The iButton is a small, waterproof, and shock-resistant device that can be used
...
   stringify -> string:
-    return "IButton-$(%016x id)"
+    return "IButton-$(%016x id_)"
+
+  /**
+  Writes the given $id to the iButton.
+
+  The device must have a changeable ID. A typical device would be the RW1990.
+
+  Depending on the version, the ID bits must be written in inverse order.
```
Comment (erikcorry) [inline src/ibutton.toit:85]: The version of what?

**Review by erikcorry — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (floitsch) [reply README.md]: It's not the RW1990 that can modify the ibuttons, but the RW1990 is an iButton device with a changeable ID.
I updated the description.
done.

File: src/ibutton.toit (hunk 3 above)
Comment (floitsch) [reply src/ibutton.toit]: Of the RW1990. I updated the doc.
done.

**Pull request merged**